configs/orchestrator/values.yaml

```diff
@@ -11,25 +15,39 @@ helm:
   version: 3.5.1
 
 sshKey:
-  public:
-  private:
+  # Copy your SSH Key content here.
+  public: ssh-rsa AAAA.YOUR_PUBLIC_KEY
+  private: |
+    -----BEGIN OPENSSH PRIVATE KEY-----
+    YOUR_PRIVATE_KEY
```

Comment:
Why is the private part needed? This seems like a bad idea.

Reply:
Updated the README so that user can provide that using a flag and don't have to save it to a file.

Reply:
My question still stands. Why is the private key needed? I'm not sure I understand the full architecture, but it seems this is uploading the private key of the user to a kubernetes cluster, is that correct?

Reply:
A user can choose to create a temporary key per deployment. Yes this is uploading the key to the (orchestrating) cluster in the form of Kubernetes secret which will deploy other (benchmarking) clusters.

Reply:
In that case, I think the documentation should explain what's happening with the key and should show an example of creating a special key for the cluster, instead of showing the example of using the default SSH key.

Reply:
done, PTAL.